lib/atlassian-jwt-authentication/filters.rb

```diff
@@ -55,11 +55,14 @@ def on_add_on_installed
           if user[:username].present? && user[:display_name].present? &&
               user[:uuid].present? && user[:type].present? && user[:type] == 'user'
 
-            jwt_user = current_jwt_auth.jwt_users.where(user_key: user[:uuid]).first
-            JwtUser.create(jwt_token_id: current_jwt_auth.id,
```

Comment:
You're not storing the jwt_token_id anymore?

Reply:
`jwt_token_id` will be set thanks to referencing via `current_jwt_auth.jwt_users`.